build-tools/packages/build-cli/README.md

```diff
@@ -21,7 +21,7 @@ $ npm install -g @fluid-tools/build-cli
 $ flub COMMAND
 running command...
 $ flub (--version|-V)
-@fluid-tools/build-cli/0.11.0
+@fluid-tools/build-cli/0.12.0
```

Comment:
Technically unrelated; I'll follow up with a separate PR to make sure these updates are done as part of the versioning process.